doc-Monitoring_Alerts_and_Reporting/topics/Integrating_Prometheus_Alerts.adoc

```diff
@@ -0,0 +1,130 @@
+. Adding OpenShift Containers Providers and Prometheus alert endpoints in {product-title_short}.
+
+==== Deploying and Configuring Prometheus on OpenShift
+
+. Deploy Prometheus on OpenShift by following the steps covered in link:https://docs.openshift.com/container-platform/3.7/install_config/cluster_metrics.html#openshift-prometheus[Prometheus on OpenShift Container Platform].
+. Add Prometheus to an OpenShift cluster and configure alert definitions. See the example code block below for configuring alerts (currently must be done on the Prometheus side):
++
+----
+$ oc edit configmap -n openshift-metrics prometheus
+# Supported annotations:
+# severity: ERROR|WARNING|INFO. defaults to ERROR.
+# miqTarget: ContainerNode|ExtManagementSystem, defaults to ContainerNode.
+# miqIgnore: "true|false", should ManageIQ pick up this alert, defaults to true.
+  prometheus.rules: |
+    groups:
+    - name: example-rules
+      interval: 30s # defaults to global interval
+      rules:
+      - alert: "NodeDown"
+        expr: up{job="kubernetes-nodes"} == 0
+        annotations:
+          miqTarget: "ContainerNode"
+          severity: "ERROR"
+          url: "https://www.example.com/node_down_fixing_instructions"
+          message: "Node {{$labels.instance}} is down"
+      - alert: "TooManyRequests"
+        expr: rate(authenticated_user_requests[2m]) > 12
+        annotations:
+          miqTarget: "ExtManagementSystem"
+          severity: "ERROR"
+          url: "https://www.example.com/too_many_requests_fixing_instructions"
+          message: "Too many authenticated requests"
+
+----
++
+. Reload the Prometheus configuration. You can do this by deleting the Prometheus pod, then redeploying with the new configuration.
+
+==== Assigning Prometheus Alert Profiles to the Enterprise
+
+Complete the following procedure to assign Prometheus alert profiles to the enterprise using the {product-title_short} user interface.
+
+[NOTE]
+====
+Both Node and Provider alert profiles are created automatically during the installation, so it is not required to create these profiles.
+====
+
+. Navigate to menu:Control[Explorer], then click *Alert Profiles* in the accordion menu.
+. Click to expand *Node Alert Profiles*, then click *Prometheus Node Profile*. 
+. Click image:1847.png[Configuration] (*Configuration*), then click image:1851.png[] (*Edit assignments for this Alert Profile*). Assign the profile to *The Enterprise* from the *Assign To* list.
+. Click *Save*.
+. Click to expand *Provider Alert Profiles*, then click *Prometheus Provider Profile*. 
+. Click *Configuration*, then click *Edit assignments for this Alert Profile*. Assign the profile to *The Enterprise* from the *Assign To* list.
+. Click *Save*.
++
+[NOTE]
+====
+Alerts triggered before assigning the profiles to the enterprise will not appear in {product-title_short} at all.
+====
++
+
+
+==== Adding OpenShift Containers Providers and Prometheus Alert Endpoints
```

Comment:
Adding OpenShift Containers Provider with a Prometheus Alert Endpoint

Reply:
This while section should be under the general endpoints section... and maybe linked from here...

Reply:
sorry, i don't understand. could you clarify this a bit more please. you mean this entire procedure/section should be moved under an existing general endpoints section and no need to have this section Adding an OpenShift Containers Provider with Prometheus Alert Endpoints here? perhaps, update and link section - https://github.com/ManageIQ/manageiq_docs/blob/6ef2bef2d9cfddbceb5e1617ccacae626cb15fe5/doc-Managing_Providers/topics/Editing_a_Containers_Provider.adoc - from here as you've commented below?

Reply:
Exactly , we already have that general endpoint section. It is a little out of sync though.
You can see there we have:
"5. Under Endpoints in the Default tab, edit the following as required:"
Remained unchanged.

"6. Under Endpoints in the Hawkular tab, configure the following for Hawkular capacity and utilization metrics collection:"

That tab in the next version is actually called "Metrics" and you can choose Hawkular OR Prometheus

There should be another section named
"6. Under Endpoints in the Alerts tab, configure the following for Prometheus alerting from the cluster"
...

Just like the existing sections for the new alert endpoint


Hope that helps.

Reply:
@moolitayer thanks for the clarification, understood. 
In that case, 
1. I'll apply updates/feedback from here (mainly in relation to the steps about Trusted CA Certificate and Token) to section https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.6-beta/html-single/managing_providers/#adding_openshift_provider in the Managing Provider's guide and link that section from under 9.1.3. Adding an OpenShift Containers Provider with a Prometheus Alert Endpoint as the third part of 9.1. Configuring Prometheus Alerts in CloudForms (instead of linking the Editing a Containers Provider section) which I think would be a better fit as part of the end-to-end steps of Configuring Prometheus Alerts in CF). 
2. Then, per your above comment - I'll also update the section https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.6-beta/html-single/managing_providers/#editing_a_containers_provider to include the missing step: "7. Under Endpoints in the Alerts tab, configure the following for Prometheus alerting from the cluster"
...  What do you think?
3. Also, I know it's not directly related to this PR/topic, but while I'm at it - wondering if Hawkular is now completely gone and removed from the Metrics dropdown? So that I can update the procedure and remove Hawkular references accordingly.
Thanks, -Suyog

Reply:
Yes the "Hawkular" tab is completely gone, we now have a "Metrics" tab with two options "Prometheus" and "Hawkular"

cc @yaacov

@suyogsainkar Do have a running 5.9? ping me if you want to take a look at a running system

Reply:
Thanks @moolitayer. Yes, I have access to a running 5.9 (version 5.9.0.9.20171115202245_7429f75), though I'll soon be updating to the latest available build. I can see the Metrics tab with the two options Prometheus and Hawkular. However I wasn't sure if there's any change with the Hawkular option in there - got confused with the removal of the middleware/hawkular provider in CF, which is not related to this. Thanks for confirming.

Reply:
![ocp-add](https://user-images.githubusercontent.com/6856557/34705516-aabdd988-f54c-11e7-82eb-0d366f4b3f3a.png)